src/pages/Information/Information.test.jsx

```diff
@@ -26,7 +26,7 @@ describe("Information", () => {
     const wrapper = setup();
     const headingDisplay = findByTestAttribute(wrapper, "heading-display");
     test("renders heading display", () => {
-      expect(headingDisplay.length).toBe(3);
+      expect(headingDisplay.length).toBe(18);
```

Comment:
🔧 Change this and the one below look like this...
```suggestion
      expect(headingDisplay.exists()).toBeTruthy();
```

Reply:
Addressed:  https://github.com/SnapITSolutions/snapit-hacktoberfest/commit/4fc501bb2d1477550a50115390ccd963961263c4